stories/utils/create-legacy-settings-wrapper.js

```diff
@@ -61,8 +61,8 @@ export default function createLegacySettingsWrapper( moduleSlug, moduleComponent
 		useEffect( () => {
 			removeAllFilters( `googlesitekit.ModuleSettingsDetails-${ moduleSlug }` );
 			addFilter(
-				'googlesitekit.ModuleSettingsDetails-tagmanager',
-				'googlesitekit.TagManagerModuleSettings',
+				`googlesitekit.ModuleSettingsDetails-${ moduleSlug }`,
+				'googlesitekit.SettingsLegacy',
```

Comment:
I added this change to fix a bug with the legacy settings wrapper which was preventing it from working properly with the other modules 😄